src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -268,6 +270,21 @@ public override async Task RunAsync()
             await runTask;
         }
 
+        private void ValidateHostJsonConfiguration()
+        {
+            bool IsPreCompiledApp = IsPreCompiledFunctionApp();
+            var hostJsonPath = Path.Combine(Environment.CurrentDirectory, Constants.HostJsonFileName);
```

Comment:
hostJsonPath is being calculated at multiple places. Move this to constant?

Reply:
I updated the logic to calculate the `hostJsonPath` only once. I cannot make it a constant because the value of the `Environment.CurrentDirectory` changes based on the context. If i make hostJson path constant the `Environment.CurrentDirectory` it would may end up using incorrect path.